components/cdc/tests/integrations/test_cdc.rs

```diff
@@ -1203,12 +1378,12 @@ fn test_cdc_1pc() {
                     assert_eq!(entries.entries.len(), 2);
                     let (e0, e1) = (&entries.entries[0], &entries.entries[1]);
                     assert_eq!(e0.get_type(), EventLogType::Committed);
-                    assert_eq!(e0.get_key(), k1);
-                    assert_eq!(e0.get_value(), v1);
+                    assert_eq!(e0.get_key(), k2);
```

Comment:
Why the order is changed here ?

Reply:
https://github.com/tikv/tikv/pull/12387#discussion_r860462211 

I want to restore the order of code.

Reply:
From the log, the original test key "key1" and "key2" convert from hashmap to vec(delegate.rs:sink_data), vec contains "key1" and "key2". 

Now the test key changes to "xkey1" and "xkey2", "xkey1" and "xkey2" convert from hashmap to vec(delegate.rs:sink_data), vec contains "xkey2" and "xkey1".  

So the oreder need to chagne.